docs/draft/3rd-party-api.md

```diff
@@ -0,0 +1,109 @@
+---
+title: 3rd party APIs used for core functionality that impacts things like availability/censorship and privacy
+version: 0.1.0
+status: Draft
+authors:
+---
+
+# 3rd party APIs used for core functionality that impacts things like availability/censorship and privacy
+
+## Table of Contents
+
+1. [Abstract](Abstract)
+2. [Definitions](#definitions)
+3. [Why 3rd party API can be a problem?](#why-3rd-party-api-can-be-a-problem)
+4. [3rd party APIs used by Status](#3rd-party-apis-used-by-status)
+  * [Infura](#infura)
+  * [Etherscan](#etherscan)
+  * [CryptoCompare](#cryptocompare)
+  * [Collectibles](#collectibles)
+  * [Iubenda](#iubenda)
+5. [Changelog](#changelog)
+6. [Acknowledgements](#acknowledgements)
+7. [Copyright](#copyright)
+
+## Abstract
+In this specification listed 3rd party APIs that Status functionality rely on.
+
+## Definitions
+
+| Term        | Description |
+| ------------- |-------------|
+| Fiat money    | Currency which established as money, often by government regulation, but that has no intrinsic value
+| Full node    | Any computer, connected to the Ethereum network, which fully enforces all the consensus rules of Ethereum.
+| Crypto-collectible | A cryptographically unique, non-fungible digital asset . Unlike cryptocurrencies, which require all tokens to be identical, each crypto-collectible token is unique or limited in quantity.
+
+
+## Why 3rd party API can be a problem?
+Relying on 3rd party APIs interferes with `censorship resistance` Status principle. Since we aim to avoid suppression of information it is important to reduce amount of 3rd parties crucial for app functionality.
+
+## 3rd party APIs used by Status
+
+### Infura
+
+##### What is it?
+Infura hosts a collection of own full nodes on the Ethereum network and provides an API access to the Ethereum and IPFS networks without having to run a full node.
+
+##### How Status use it?
+Status works on mobile devices and therefore can't rely on local node. So all communication to Ethereum network happens via Infura.
+
+##### Concerns
+Making http request means that user metadata leaks. Also if service hacked it can be used in various attacks, e.g. by faking returning data.
+Infura hosts on Amazon. It can fail or Amazon can cut off service or their servers crash. In this case all Status features related to Ethereum network calls will fail.
+
+
+### Etherscan
+##### What is it?
+Etherscan is a service that allows user to explore and search the Ethereum blockchain for transactions, addresses, tokens, prices and other activities taking place on Ethereum network.
+
+##### How Status use it?
```

Comment:
Can we be more specific with what type of calls we require from them at an interface level?

This will then feed into requirements for projects such as Nimbus, and other ways of achieving the same goal, such as Beam sync, etc.

Reply:
We don't have any calls to Etherscan, we create a links to it like this:
```
(defn- etherscan-url [address]
  (str "https://etherscan.io/address/" address))
```
So strictly speaking this is not an API but still a 3rdparty we rely on (same thing with Iubenda).
Should it be removed from the doc? Or maybe the doc renamed to something like "3rd-party services used by current Status app"?

Reply:
How is etherscan-url used? A user clicks on it? (or is it parsed?) So to get the capability of reading historical we rely on that 3rd party, which has X implications.

Reply:
@oskarth,
It is used in link that user can click on:
```
[link {:on-press #(.openURL react/linking (etherscan-url contract))}
        (i18n/label :t/etherscan-lookup)]
```
>So to get the capability of reading historical we rely on that 3rd party, which has X implications
We have own capability of reading historical but also give a user possibility to look at Etherscan. For example user can open transaction detail page in Status. But on that page he can chose to open same transaction on Etherscan.